src/client/utils/hooks/useQuoteCartData.ts

```diff
@@ -191,7 +191,7 @@ export const useQuoteCartData = () => {
     selectedInsuranceTypes,
   )
 
-  if (!selectedQuoteBundleVariant) throw new Error('Invalid state')
+  if (!selectedQuoteBundleVariant) return
```

Comment:
Is this a valid state now? Or what are we representing by returning `undefined`?

Reply:
replaced this with null 👍